geniza/corpus/admin.py

```diff
@@ -312,7 +312,7 @@ def get_deleted_objects(self, objs, request):
         return deletable_objects, model_count, perms_needed, protected
 
     def get_queryset(self, request):
-        return super().get_queryset(request).metadata_prefetch()
```

Comment:
This might fix the unit tests, but it will make the admin list view unusable. I think the right way to fix it is for the export code to only apply the `metadata_prefetch` when an existing queryset is not passed in.

Reply:
In the `get_queryset` method of the Document exporter, we are currently applying all prefetching. I think it should be revised so that if `self.queryset` is set we only apply the things after the `metadata_prefetch` (and document that this is the case — if a large queryset is passed in, metadata prefetching should be applied before exporting).

Reply:
The admin list view still works for me, if that's what I think it means? But I see what you mean about prefetching going on even when .queryset is set. 

This is current get_queryset in document exporter:
```python
def get_queryset(self):
    return (
        super()
        .get_queryset()
        .metadata_prefetch()
        .prefetch_related("secondary_languages", "log_entries")
        .order_by("id")
    )
```
How about this -- is this what you mean?

```python
def get_queryset(self):
    qset = self.queryset or self.model.objects.all().metadata_prefetch()
    qset = qset.prefetch_related("secondary_languages", "log_entries").order_by("id")
    return qset
```

Reply:
I'm assuming that that secondary prefetching is necessary no matter whether an initial queryset was set or not.